src/Controls/src/Core/Platform/Android/CustomHoverGestureRecognizer.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Android.Views;
+using AView = Android.Views.View;
+
+namespace Microsoft.Maui.Controls.Platform.Android;
+
+internal class CustomHoverGestureRecognizer : Java.Lang.Object, AView.IOnHoverListener
```

Comment:
What do you think about just making `PointerGestureHandler` implement

```C#
Java.Lang.Object, AView.IOnHoverListener
```

Then we don't need this type.  Similar to how `DragAndDropGestureHandler` is written.